recipes/crlibm/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "crlibm" %}
+{% set version = "1.0.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 48e17981f90d69c6bb0013f68bacbe7a157de864a533d15dd196ca7e98348a35
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```

From looking at the CI logs it's using the wrong compiler, this might help (you'll need to remove the skip as well):
```suggestion
    - {{ compiler("m2w64_c") }}  # [win]
    - {{ compiler('c') }}  # [not win]
```

---

(we can ignore appveyor failing, Azure is good enough)

---

Ok, thats good to know :-)
The windows build on Azure now completed!